### ISSUE TITLE
sandbox: when a bot plays against itself it receives same scores for both X and O moves

### ISSUE THREAD

**Issue by stefanb** [ogrodje/tttm#12]
When a game server asks a game bot to play against itself the scoring is messed up, giving them both exactly the same result:
<img width="1022" alt="image" src="https://github.com/user-attachments/assets/991895f0-5a61-4f45-94b0-6aba65b71721">
<img width="992" alt="image" src="https://github.com/user-attachments/assets/2b0a26b0-5500-47ed-acc4-668e90448373">

**Comment by otobrglez** [comment]
Thank you, @stefanb, for reporting this.

I am aware of this bug, and as far as I know, It only affects the sandbox, and the root cause is in the ID that is extrapolated from the hostname. I need to investigate further and fix it.

**Comment by stefanb** [comment]
Yeah, the computed id would also need to include x or o symbol and path and port... So basically whole base url + symbol.

Obviously this can differ from what is shown in the table - the table can still show just the hostname or better: some other short, unique player descriptor, eg player's `name` 😄
